scripts/install/lidarr.sh

```diff
@@ -8,26 +9,47 @@
 #   changes/dates in source files. Any modifications to our software
 #   including (via compiler) GPL-licensed code must also be made available
 #   under the GPL along with build & install instructions.
-user=$(cut -d: -f1 < /root/.master.info)
-ip=$(ip route get 1 | sed -n 's/^.*src \([0-9.]*\) .*$/\1/p')
-distribution=$(lsb_release -is)
-version=$(lsb_release -cs)
-#shellcheck source=sources/functions/mono
-. /etc/swizzin/sources/functions/mono
-mono_repo_setup
-apt_install libmono-cil-dev libchromaprint-tools
 
-echo_progress_start "Downloading Lidarr release and extracting"
-cd /home/${user}/
-wget -O lidarr.tar.gz -q $(curl -s https://api.github.com/repos/Lidarr/Lidarr/releases | grep linux.tar.gz | grep browser_download_url | head -1 | cut -d \" -f 4)
-tar xf lidarr.tar.gz
-rm -rf lidarr.tar.gz
-chown -R ${user}: /home/${user}/Lidarr
-echo_progress_done "Lidarr downloaded and extracted"
+install() {
+    #shellcheck source=sources/functions/users
+    . /etc/swizzin/sources/functions/users
+    user="$(_get_master_username)"
+    apt_install mediainfo sqlite3 libchromaprint-tools
+
+    echo_progress_start "Downloading source files"
+
+    urlbase="https://lidarr.servarr.com/v1/update/develop/updatefile?os=linux&runtime=netcore"
```

Comment:
this should now be `master` not `develop`

Reply:
```suggestion
    urlbase="https://lidarr.servarr.com/v1/update/master/updatefile?os=linux&runtime=netcore"
```